Gifski/DraggableFile.swift

```diff
@@ -55,17 +55,22 @@ final class DraggableFile: NSImageView {
 	}
 
 	func animateScale() {
-		let springAnimation = CASpringAnimation(keyPath: "transform.scale")
+		let springAnimation = CASpringAnimation(keyPath: #keyPath(CALayer.transform))
+
+		var tr = CATransform3DIdentity
+		tr = CATransform3DTranslate(tr, bounds.size.width / 2, bounds.size.height / 2, 0)
+		tr = CATransform3DScale(tr, 2.0, 2.0, 1)
+		tr = CATransform3DTranslate(tr, -bounds.size.width / 2, -bounds.size.height / 2, 0)
```

Comment:
Would be nice to move this logic into a `animateScaleCenter`-kinda method as an extension on `CALayer`, with a `scale` parameter.

Reply:
I've changed it a little bit to animate also the y position. So how could I move this to `animateScaleCenter`?

Reply:
Call it `animateScaleMove()` or something and handle both scaling and moving.